### ISSUE TITLE
Replay recorder needs a lot more work

### ISSUE THREAD

**Issue by polina4096** [polina4096/apex#8]
As the title says, the replay recorder mostly works, but it really needs a lot more polish to be generally usable. The feature is very low priority, so it is not worth the time which it requires at the moment.

The user interface can be improved, settings to configure ffmpeg location and it's detection/downloading can be implemented (consider using `ffmpeg-sidecar`?). Configurable interface for rendering parts or the whole play can be implemented. Also this feature is kinda blocked by local replays and scores, which we don't have at the moment. Right now it renders a perfect auto play.

A lot of work, consider doing it much later.
